tests/testsuite/standard_lib.rs

```diff
@@ -335,6 +269,81 @@ fn depend_same_as_std() {
         None => return,
     };
 
+    // Our mock sysroot requires a few packages from crates.io, so make sure
+    // they're "published" to crates.io. Also edit their code a bit to make sure
+    // that they have access to our custom crates with custom apis.
+    Package::new("registry-dep-using-core", "1.0.0")
```

Comment:
With the addition of these crates in the sysroot, this is now not required, right? In theory we shouldn't need to download anything from the registry to build the sysroot crates.

Reply:
They're required for exactly this test because it's explicitly testing a project that happens to depend on all our vendored libraries, but, not vendored. Wasn't sure what to do with it. I could probably make the test have it use that directory as its own vendor if we don't want this nasty duplication.